vendor/bat-native-ledger/src/bat/ledger/internal/test/mock_ledger_client.cc

```diff
@@ -55,13 +55,4 @@ void MockLedgerClient::SavePublisherState(
   handler->OnPublisherStateSaved(ledger::Result::OK);
 }
 
-void MockLedgerClient::LoadURL(const std::string& url,
-                               const std::vector<std::string>& headers,
-                               const std::string& content,
-                               const std::string& contentType,
-                               const ledger::UrlMethod method,
-                               ledger::LoadURLCallback callback) {
-  callback(true, "{}", {});
-}
-
```

Comment:
This mock was providing a 'default' mock logic which made no sense, as it prevents mocking it. I've repurposed it to a true MOCK method, and all unit tests pass (including mine which were failing due to this).